src/SoundContext.cpp

```diff
@@ -64,17 +65,18 @@ void SoundContext::init() {
 }
 
 void SoundContext::addSoundToLoad(SoundInfo* info) {
-    soundsToLoad_->push_back(info);
+    (*soundsToLoad_)[info->id_] = info;
 }
 
-FMOD::Sound* SoundContext::getSound(const std::string& id) {
-    return sounds_[id];
-}
-
-Channel* SoundContext::playSound(FMOD::Sound* sound) const {
+Channel* SoundContext::playSound(const std::string& id) const {
     FMOD::Channel* channel;
     try {
-        auto result = system_->playSound(sound, 0, false, &channel);
+        auto result =
+            system_->playSound(getInstance()->getSound(id), 0, true, &channel);
```

Comment:
I'm pretty sure this is useless.
```suggestion
            system_->playSound(getSound(id), 0, true, &channel);
```

Reply:
getInstance está ahí porque si no, no me deja compilar el código siquiera